core/bin/zksync_tee_prover/src/config.rs

```diff
@@ -22,10 +24,12 @@ pub(crate) struct TeeProverConfig {
     pub max_retries: usize,
     /// Initial back-off interval when retrying recovery on a retriable error. Each subsequent retry interval
     /// will be multiplied by [`Self.retry_backoff_multiplier`].
+    #[serde_as(as = "DurationSecondsWithFrac")]
     pub initial_retry_backoff: Duration,
```

Comment:
We usually load timeouts like seconds or milliseconds (and add corresponding postfix, e.g. `_sec`), so that the value is predictably stored as integer, and add a convenience getter, e.g. `fn initial_retry_backoff(&self) -> Duration { Duration::from_secs(self.initial_retry_backoff_sec) }`.

Reply:
I prefer to introduce abstractions (like `Duration`) as early as possible to avoid the need for verbose intermediary getters that act as translators between abstraction layers (`u64` <--> `Duration`), but that's fine – I can stick to the conventions we have here. 🫡